JavaScript/main.js

```diff
@@ -1,8 +1,103 @@
+  // String: greeting
+  {
+    let userName = prompt(`Будь ласка, введіть своє ім'я:`);
+    alert(`Привіт `+ userName +`! Як справи?`);
+  }
+
+  // String: gopni4ek
+  {
+    let inputString = prompt("Будь ласка, введіть рядок:");
+
+    // inputString.replaceAll(",", "блін, ") що тут не так?
+
+    // це чат gpt допоміг
+    let modifiedString = inputString.split(",").join("блін, ");
+
+    alert("Результат: " + modifiedString);
+  }
+
+  // String: capitalize;
+  {
+    let str = "cANBerRa";
+    let result = str.charAt(0).toUpperCase() + str.slice(1).toLowerCase();
+    console.log(result);
+  }
+
+  // String: word count
+  {
+    let inputString = prompt("Будь ласка, введіть рядок:");
+    let wordsArray = inputString.split(" ");
+    let wordCount = wordsArray.length;
+
+    console.log("Кількість слів у рядку:", wordCount);
+  }
+
+  // String: credentials
+  {
+    String.prototype.capitalize = function () {
```

Comment:
Don't use code that you don't understand 
Please check my solution 
https://github.com/cos1715/FS2/blob/lesson-12/hw.js